orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -412,13 +419,24 @@ def _get_exported_model(self, input_schema, *inputs, **kwargs):
                     "input_names": self._input_info.names,
                     "output_names": output_names,
                     "opset_version": ortmodule.ONNX_OPSET_VERSION,
-                    "do_constant_folding": False,
+                    "do_constant_folding": self._runtime_options.exporter_options.do_constant_folding,
                     "training": self._export_mode,
-                    "dynamic_axes": self._input_info.dynamic_axes,
                     "verbose": self._debug_options.logging.log_level < LogLevel.WARNING,
-                    "export_params": False,
-                    "keep_initializers_as_inputs": True,
+                    "export_params": self._runtime_options.exporter_options.export_params,
+                    "keep_initializers_as_inputs": self._runtime_options.exporter_options.keep_initializers_as_inputs,
                 }
+
+                if not self._runtime_options.exporter_options.use_static_shapes:
+                    required_export_kwargs["dynamic_axes"] = self._input_info.dynamic_axes
```

Comment:
nit: you can assign the value on line 417 itself no need to add this here
some_expression if condition else other_expression